frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -0,0 +1,425 @@
+  align-items: center;
+`
+
+const StyledButton = styled(ButtonLink)`
+  width: 220px;
+`
+
+const Platform = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 16px;
+`
+
+const DetailSection = styled.section`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+`
+
+const Description = styled.p`
+  ${{ ...theme.typography.body2 }}
+  color: ${theme.custom.colors.darkGray2};
+  margin: 0;
+  white-space: pre-line;
+`
+
+const StyledPlatformLogo = styled(PlatformLogo)`
+  height: 26px;
+`
+
+const OnPlatform = styled.span`
+  ${{ ...theme.typography.body2 }}
+  color: ${theme.custom.colors.black};
+`
+
+const Topics = styled.section`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+`
+
+const TopicsList = styled.ul`
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+  margin: 0;
+  gap: 8px;
+`
+
+const InfoItems = styled.section`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+`
+
+const InfoItem = styled.div`
+  display: flex;
+  align-items: flex-start;
+  gap: 16px;
+  align-self: stretch;
+  ${{ ...theme.typography.body2 }}
+  color: ${theme.custom.colors.silverGrayDark};
+
+  svg {
+    width: 20px;
+    height: 20px;
+    flex-shrink: 0;
+  }
+`
+
+const InfoLabel = styled.div`
+  width: 85px;
+  flex-shrink: 0;
+`
+
+const InfoValue = styled.div`
+  ${{ ...theme.typography.body2 }}
+  color: ${theme.custom.colors.black};
+`
+
+type LearningResourceExpandedProps = {
+  resource?: LearningResource
+  imgConfig: EmbedlyConfig
+}
+
+const ImageDisplay: React.FC<{
+  resource?: LearningResource
+  config: EmbedlyConfig
+}> = ({ resource, config }) => {
+  if (resource?.resource_type === "video" && resource?.url) {
+    return (
+      <EmbedlyCard
+        aspectRatio={config.width / config.height}
+        url={resource?.url}
+        embedlyKey={config.key}
+      />
+    )
+  } else if (resource?.image) {
+    return (
+      <Image
+        src={resourceThumbnailSrc(resource?.image, config)}
+        aspectRatio={config.width / config.height}
+        alt={resource?.image.alt ?? ""}
+      />
+    )
+  } else {
+    return <SkeletonImage variant="rectangular" height={220} />
+  }
+}
+
+const ResourceTitle = ({ resource }: { resource?: LearningResource }) => {
+  if (!resource) {
+    return <Skeleton variant="text" height={20} width="66%" />
+  }
+  return (
+    <Typography variant="h5" component="h2">
+      {resource.title}
+    </Typography>
+  )
+}
+
+const ResourceDescription = ({ resource }: { resource?: LearningResource }) => {
+  if (!resource) {
+    return (
+      <>
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="33%" />
+      </>
+    )
+  }
+  return <Description>{resource.description}</Description>
```

Comment:
Some descriptions contain HTML - do we need to display as HTML. Not a good idea of course due to the XSS vulnerability, so ideally sanitized on the backend. Others (most so far) are plain text. We are handling `\n` newlines with CSS, but not rendering HTML in this PR.

**Example:** 
https://mitopen-rc.odl.mit.edu/api/v1/learning_resources/14387/

Renders as:
<img width="591" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/307bbe26-d4e3-4da9-a0e4-6cee77e43f8f">

Reply:
Matt is working on cleaning these up. See 
- https://github.com/mitodl/hq/issues/4355

The plan is the render the raw HTML—`dangerouslySetInnerHtml={{ __html: description }}` should be fine since. the HTML is being sanitized on the backend.